addons/test_mail/tests/test_performance.py

```diff
@@ -47,6 +47,12 @@ def setUp(self):
         super(BaseMailPerformance, self).setUp()
         # patch registry to simulate a ready environment
         self.patch(self.env.registry, 'ready', True)
+        self.flush_tracking()
+
+    def flush_tracking(self):
+        """ Force the creation of tracking values. """
+        self.env['base'].flush()
+        self.cr.flush()
```

Comment:
Looks a lot like a "flush the whole world to make it work".

Can't we just flush `mail.tracking.value` ?

Reply:
It is a copy of what is done in mail test class. This was done by the framework I think.